File: snippets/py/array/insert/insert.py
array = [3, 2, 1]
index = 1  # 0: first item, 1: second item...
item = 100
print(array)  # [3, 2, 1]
array = array[0:index] + [item] + array[index:]  # ≡
print(array)  # [3, 100, 2, 1]

File: snippets/js/array/insert/insert.js
var array = [3, 2, 1];
var index = 1;  // 0: first item, 1: second item...
var item = 100;
console.log(array);  // [ 3, 2, 1 ]
array.splice(index, 0, item);  // ≡
console.log(array);  // [ 3, 100, 2, 1 ]